### ISSUE TITLE
Send to folder does not use VFAT safe names

### ISSUE THREAD

**Issue by vossad01** [gpodder/gpodder#282]
When a filename contains characters that are not valid on a destination VFAT target, send to fails with the following exception:

```
1492734407.836906 [gpodder.log] ERROR: Uncaught exception: Traceback (most recent call last):
  File "/home/user/dev/gpodder/src/gpodder/gtkui/main.py", line 1750, in save_episodes_as_file
    shutil.copyfile(copy_from, copy_to)
  File "/usr/lib/python3.5/shutil.py", line 115, in copyfile
    with open(dst, 'wb') as fdst:
OSError: [Errno 22] Invalid argument: '/media/user/0123-4567/Podcasts/Episode 84: Ur Fave is Problematic.mp3'

Traceback (most recent call last):
  File "/home/user/dev/gpodder/src/gpodder/gtkui/main.py", line 1750, in save_episodes_as_file
    shutil.copyfile(copy_from, copy_to)
  File "/usr/lib/python3.5/shutil.py", line 115, in copyfile
    with open(dst, 'wb') as fdst:
OSError: [Errno 22] Invalid argument: '/media/user/0123-4567/Podcasts/Episode 84: Ur Fave is Problematic.mp3'
```

This was also reported by @dannn-o in unrelated issue #279 .

This is important because a lot of MP3 players will expose themselves as USB Mass Storage with msdos formatting.

**Podcast for reproduction:** http://rss.acast.com/callyourgirlfriend (most every recent episodes includes a colon (**:**) which will trigger the failure.

## Workaround
Send to a supported target volume first then copy to the ultimate destination counting on the OS to handle sanitizing the file name.  (This works on Linux Mint Cinnamon anyway).

**Comment by vossad01** [comment]
Rather the troubling gPodder with inspecting the target file system, I am inclined to just do a `try ... except ...`

If the original name does not match, then replace the following characters `" * / : < > ? \ |` (in the filename, not the entire path) with an underscore (`_`).  Source: [[1]](https://msdn.microsoft.com/en-us/library/aa365247.aspx#naming_conventions), [[2]](https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations)

**Comment by vossad01** [comment]
Fixed by #283

**Comment by vossad01** [comment]
This was caused #189.  Prior to that, the OS's file system dialog would have disallowed the unsupported filename.
